### PR TITLE
Fix embedding for tensorflow

### PR DESCRIPTION
**Pull request by Hzaatiti**

### CODE REVIEW

File: geomstats/_backend/pytorch/__init__.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
     lt as less,
     matmul,
     max as amax,
+    mean,
```
Comment (ninamiolane) [inline geomstats/_backend/pytorch/__init__.py:42]: Since numpy's and torch's API of mean are different (`axis` is `dim` in torch) we should keep the lines below. Or use https://github.com/geomstats/geomstats/pull/540

If you are interested in using this, let me know and I'll merge the PR.

File: geomstats/learning/frechet_mean.py [hunk 2]
```diff
@@ -167,7 +167,8 @@ def _ball_gradient_descent(points, metric, weights=None, max_iter=32,
 
         iteration = 0
         convergence = math.inf
-        barycenter = points.mean(0, keepdims=True) * 0
+        barycenter = gs.mean(points, axis=0, keepdims=True)
```
Comment (ninamiolane) [inline geomstats/learning/frechet_mean.py:170]: How does this work in pytorch, since `axis` is not a parameter of `torch.mean`https://pytorch.org/docs/1.4.0/torch.html?highlight=torch%20mean#torch.mean?

File: tests/test_prepare_graph_data.py [hunk 3]
```diff
@@ -58,7 +57,6 @@ def test_loss(self):
         self.assertAllClose(loss_value[0], expected_loss, rtol=1e-3)
         self.assertAllClose(gs.squeeze(loss_grad), expected_grad, rtol=1e-3)
 
-    @geomstats.tests.np_and_pytorch_only
     def test_embed(self):
         """Test embedding function."""
         embeddings = self.embedding_class.embed(self.karate_graph)
```
Comment (ninamiolane) [inline tests/test_prepare_graph_data.py:62]: NIT (not so important for the tests, but more important for the core code): we should not use `embedding_class` to call the OBJECT from the class HyperbolicEmbedding, because it is precisely an object and not a class.

**Comment by ninamiolane** [comment]
Good to merge once the conflicts are solved ! (+ small questions on the PR).
